Source/WebKit/UIProcess/API/APIWebsitePolicies.h

```diff
@@ -152,6 +155,7 @@ class WebsitePolicies final : public API::ObjectImpl<API::Object::Type::WebsiteP
     // FIXME: replace most or all of these members with a WebsitePoliciesData.
     bool m_contentBlockersEnabled { true };
     HashMap<WTF::String, Vector<WTF::String>> m_activeContentRuleListActionPatterns;
+    Vector<WTF::String> m_disabledContentRuleListIdentifiers;
```

Comment:
It would be helpful to prevent bad code design and forgetting checks and moves to combine this with m_contentBlockersEnabled and replace them both with an Expected<ContentBlockersEnabled, MemoryCompactRobinHoodHashSet<String>>

Reply:
To clarify, do you mean the following?
1. Replace the `bool` flag with a named enum class, `ContentBlockersEnabled`.
2. Use a single `Expected<ContentBlockersEnabled, MemoryCompactRobinHoodHashSet>` in place of both `m_contentBlockersEnabled` and `m_disabledContentRuleListIdentifiers`